### PR TITLE
Upgrade jenkins to 2.452.3

### PR DESCRIPTION
**Pull request by ZiLiangYeoh**
upgrade jenkins to 2.452.3 with DAI version 7.4.0-4 updates

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,8 +15,9 @@
         <revision>0.0.1</revision>
         <changelist>-SNAPSHOT</changelist>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>2.277.1</jenkins.version>
-        <java.level>8</java.level>
+        <jenkins.version>2.452.3</jenkins.version>
+        <java.level>11</java.level>
```
Comment (jonesbusy) [inline pom.xml]: java.level is deprecated. Should be removed

File: pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.38</version>
+        <version>4.52</version>
```
Comment (jonesbusy) [inline pom.xml]: Why not directly upgrade to latest parent (4.86) ?

File: pom.xml (hunk 1 above)
Comment (ZiLiangYeoh) [reply pom.xml]: Thanks!

File: pom.xml (hunk 2 above)
Comment (ZiLiangYeoh) [reply pom.xml]: Make sense, thanks!

**Pull request closed without merging**